halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/openstack/OpenstackCommandProperties.java

```diff
@@ -17,7 +21,19 @@
 
   static final String USER_DATA_FILE_DESCRIPTION = "User data passed to Heat Orchestration Template. Replacement of tokens supported, see http://www.spinnaker.io/v1.0/docs/target-deployment-configuration#section-openstack for details.";
 
+  static final String HEAT_TEMPLATE_LOCATION_DESCRIPTION = "The location of your heat template file. (Replacing the Heat template is not recommended)";
+
   static final String LBAAS_POLL_TIMEOUT_DESCRIPTION = "Time to stop polling octavia when a status of an entity does not change. Default 60.";
 
   static final String LBAAS_POLL_INTERVAL_DESCRIPTION = "Interval in seconds to poll octavia when an entity is created, updated, or deleted. Default 5.";
+
+  static final String INSTANCE_TYPE_DESCRIPTION = "The instance type for the baking configuration.";
+
+  static final String SOURCE_IMAGE_ID_DESCRIPTION = "The source image ID for the baking configuration.";
+
+  static final String SSH_USER_NAME_DESCRIPTION = "The ssh username for the baking configuration.";
+
+  static final String REGION_DESCRIPTION = "The region for the baking configuration.";
```

Comment:
@edwinavalos Does this argument support multiple regions separated by commas? If so it's likely worth calling that  out here in the description. If not, thank you for educating me.